libraries/standard/mqtt/src/mqtt_lightweight.c

```diff
@@ -950,15 +950,15 @@ static MQTTStatus_t readSubackStatus( size_t statusCount,
         /* MQTT 3.1.1 defines the following values as status codes. */
         switch( subscriptionStatus )
         {
-            case 0x00:
-            case 0x01:
-            case 0x02:
+            case MQTTQoS0:
+            case MQTTQoS1:
+            case MQTTQoS2:
```

Comment:
Please revert this, this is a MISRA 10.3 violation
```suggestion
            case 0x00:
            case 0x01:
            case 0x02:
```

Reply:
Hmmm..That's unfortunate

Reply:
Have added `MQTT_QOS_X` macros to make these case statements expressive

Reply:
I think the macros are unnecessary, it already seems clear what the values mean, especially due to the log statement that's there.